### DISCUSSION TITLE
Questão TI21#4

### DISCUSSION THREAD

**Discussion by paulocafee** [paulowh/TI21#10] [Q&A] [answered]
Explique o conceito de "escopo" em JavaScript.

**Comment by Lucasaraujo2** [comment]
O escopo em JavaScript se refere à visibilidade de variáveis e funções dentro de um programa.
Quando uma variável ou função é declarada em um determinado escopo, ela só pode ser acessada dentro daquele escopo ou de escopos internos a ele.

**Comment by Malvadaokkj** [comment, marked as answer]
Existe dois escopos, um é o global e um é o local

O escopo global seria o codigo ao todo 
E o local seria o codigo dentro da função
